### DISCUSSION TITLE
How do I get all elements in a schema in python SDK?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#18375] [Q&A and General discussion]
How do I get all elements in a schema in python SDK?
I want to get all elements (without fitter by primary key) for debug purposes.

**Comment by yhmo** [comment]
Use an expression that can filter all entities:
query(expr="id > 0", output_fields=["a", "b", "c"])

But it is not recommended to use query() to fetch all entities, because the performance is bad, we need to re-construct the api.
